code/clue.js

```diff
@@ -1,59 +1,353 @@
+// OBJECTS FOR ALL THE SUSPECTS
 
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
-  color: 'green',
+  color: '#7FAE79',
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur',
+  favouriteWeapon: "knife"
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: '#806E8A',
+  description: '',
+  age: 60,
+  image: 'assets/plum.png',
+  occupation: 'Professor',
+  favouriteWeapon: "poison"
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: '#AE4646',
+  description: '',
+  age: 25,
+  image: 'assets/scarlet.png',
+  occupation: 'Actress',
+  favouriteWeapon: "bat"
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: '#689DAD',
+  description: 'Mother of Miss Scarlet',
+  age: 50,
+  image: 'assets/peacock.png',
+  occupation: 'Lawyer',
+  favouriteWeapon: "rope"
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: '#D4A04C',
+  description: '',
+  age: 30,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel',
+  favouriteWeapon: "trophy"
+}
+
+const mrsWhite = {
+  firstName: 'Jennie',
+  lastName: 'White',
+  color: 'white',
+  description: '',
+  age: 75,
+  image: 'assets/white.png',
+  occupation: 'retired',
+  favouriteWeapon: "candlestick"
+}
+
+// OBJECTS FOR ALL THE WEAPONS
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  weight: 10,
+  id: 'rope'
+}
+
+const knife = {
+  name: 'Knife',
+  weight: 6,
+  id: 'knife'
+}
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 8,
+  id: 'candlestick'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 3,
+  id: 'dumbbell'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const poison = {
+  name: 'Poison',
+  weight: 2,
+  id: 'poison'
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 8,
+  id: 'axe'
+}
+
+const bat = {
+  name: 'Bat',
+  weight: 5,
+  id: 'bat'
+}
+
+const trophy = {
+  name: 'Trophy',
+  weight: 5,
+  id: 'trophy'
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 4,
+  id: 'pistol'
+}
+
+
+//Grouped into arrays
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+
+
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio'
 ]
 
-const weapons = []
+let mystery = {
+  killer: 'who',
+  weapon: 'what',
+  room: 'where'
+}
```

Comment:
Interesting.  I would suggest using `null` values here, unless these initial values are used somewhere for another purpose than checking if they have been set to something else.